### ISSUE TITLE
New Component: Notification / Toast

### ISSUE THREAD

**Issue by vongrippen** [microsoft/fluentui#865]
In many apps, onscreen notifications of background events are a necessity. Currently there is not a suitable component for rendering these notifications. I'd like to see a component similar to the Office 365 notification seen when an email is received in OWA.
<img width="484" alt="notification" src="https://cloud.githubusercontent.com/assets/273675/22187863/1d308112-e0d3-11e6-864a-160b53fbad0c.png">

**Comment by micahgodbolt** [comment]
similar to toast #5479

**Comment by micahgodbolt** [comment]
will probably be dependent on #5322

**Comment by dbjorge** [comment]
I just came across [this blog post](https://medium.com/@sheribyrnehaber/designing-toast-messages-for-accessibility-fb610ac364be) on accessibility considerations for toast notifications; I am not familiar with the author, but I thought it was well-written and comprehensive. Good reference as you're considering the design of this component :)

**Comment by DuncanFisher** [comment]
Is there any update on Fabric Toast?

**Comment by wsywq** [comment]
Also want to know Toast component.

**Comment by kamil7x** [comment]
Any updates on this one? It would be really nice to have

**Comment by nbelyh** [comment]
Any updates? would really love to see this component added 😄

**Comment by dospolov** [comment]
can't believe Fluent does not have toasts/notification. Any UI kit does

**Comment by denis-sokolov** [comment]
FluentUI has [a MessageBar](https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar), though it does not include global positioning helpers.
![image](https://user-images.githubusercontent.com/113721/151420101-2643723f-3cbe-4888-9d3f-67928815c64f.png)
